src/gtktest.rs

```diff
@@ -212,7 +220,8 @@ fn main() {
         println!("key pressed: {} / {:?}", keyval, keystate);
         println!("text: {}", entry_clone.get_text().unwrap());
 
-        if keystate.intersects(modifier_type::ControlMask) {
+        use glib::translate::ToGlib;
+        if keystate.intersects(gdk::ModifierType::CONTROL_MASK.to_glib()) {
```

Comment:
We have quite the issue here: it expects a sys type, not a "normal" one. Does anyone know why?

Reply:
Because keystate was still created as ffi,
you can change `let keystate` to use https://github.com/gtk-rs/gdk/blob/master/src/event_key.rs#L19, and IMHO keyval too.